### PR TITLE
Kafka Driver Upgrading & Better  Makefile for WebAssembly Build

### PR DESCRIPTION
**Pull request by haoel**
- upgrade the Kafka Driver from 1.21.1 to 1.30.0
- New makefile target for WebAssembly build - `make wasm`

### CODE REVIEW

**Review by megaeasex — CHANGES_REQUESTED** [review]
[TASK:easegress-pr-test FAILED]megaease/easegress Pull Request 349 Deploy Test failed

File: README.zh-CN.md [hunk 1]
```diff
@@ -140,6 +140,11 @@ $ git clone https://github.com/megaease/easegress && cd easegress
 $ make
 ```
 
+> **注意事项**：
+> - 我们需要 Go 1.16+ 以上版本的编辑器
+> - 如果你需要WebAssembbly的版本，你需要运行 `make wasm`
```
Comment (localvar) [inline README.zh-CN.md:145]:
```suggestion
> - 我们需要 Go 1.16 以上版本的编译器
> - 如果需要支持 WebAssembbly 的版本，你需要运行 `make wasm` 
```

**Review by megaeasex — CHANGES_REQUESTED** [review]
[TASK:easegress-pr-test FAILED]megaease/easegress Pull Request 349 Deploy Test failed

**Review by localvar — APPROVED** [review]

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 349 Deploy Test Success

**Pull request merged**